### DISCUSSION TITLE
Add a pir to the webserver

### DISCUSSION THREAD

**Discussion by Mario-vminion** [easytarget/esp32-cam-webserver#285] [Q&A]
Hi i’m new to this kind of hard and software i see that you can add the pir motion detection to the esp32-cam how is that in combination with this webserver software … where can i find this setup or explanation

thanks

**Comment by easytarget** [comment]
A good question, and would be possibel to implement; but there is not fnctionality in the code that could do this. 

Anyw.. sorry; I am not doing any further development on this project. it is old code and somewhat obsolete. I am currently working on retiring it; I will try to link to a more actively updated fork when I do.
